plugin/common/kernels/sortScoresPerClass.cu

```diff
@@ -87,9 +87,13 @@ __launch_bounds__(nthds_per_cta)
                     // result, but will help reduce the computation because
                     // we only need to sort the mantissa part of the floating-point
                     // numbers
-                    temp_scores[targetIdx] = add_fb(score, score_shift);
-                    if (ge_fb(temp_scores[targetIdx], T_SCORE(2.f - 1.f / 1024.f)))
-                        temp_scores[targetIdx] = T_SCORE(2.f - 1.f / 1024.f);
+                    if(score_shift > 0.f){
```

Comment:
Thanks @fsx950223. Please incorporate minor formatting suggestions - `{` and `else` in newline and space between `if` and `(`